source/includes/aws/_instances.md

```diff
@@ -228,8 +228,13 @@ curl -X POST \
     "iops": 3000,
     "throughput": 250
   }],
-  "vpcId" : "vpc-0b23fed563eebe635",
-  "subnetId" : "subnet-0d4d9d813e763d403"
+  "vpcId": "vpc-0b23fed563eebe635",
+  "subnetId": "subnet-0d4d9d813e763d403",
+  "volumesToAttach": [{
+    "volumeId": "vol-1",
+    "device": "/dev/sdg",
+    "availabilityZone": "us-east-1a"
```

Comment:
do we need the availability zone? Wouldn't simplifying the request body, and doing extra work on our end be better?

Reply:
sorry, the availability zone is not actually required. Removed in https://github.com/cloudops/cloudmc-api-docs/pull/496/commits/c1f1755bdc648388153d436f5b72dffe78e7a23d